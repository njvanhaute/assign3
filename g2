0 10 9 ;
6 7 11 ;
5 9 ;
4 8 7 ; 
8 10 6 ;
11 12 2 ;
1 6 5 ;
0 6 10 ; 
0 5 3 ; 
0 4 8 ; 
0 3 2 ; 
0 1 4 ; 
