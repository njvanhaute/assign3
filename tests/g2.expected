0: 0
1: 1(0)4 3(0)2 4(0)8 5(0)3
2: 6(1)5 8(4)7 9(5)1
3: 7(6)11 10(8)6
total weight: 47
----
0: 11
1: 12(11)2
total weight: 2
----
